pywbem/cim_operations.py

```diff
@@ -1255,14 +1331,404 @@ class are to be included in the returned instances, as follows:
+          #TODO define typing for this correctly
+          OperationTimeout (term: `Uint32`):
+            Optional: If provided it defines a timeout value to be
+            sent to the WBEM Server
+            * If `Uint32` value, this value is sent to the server as the
+              proposed timeout for the operation sequence. If the client
+              waits longer than this timeout in seconds, the server may
+              terminate the operation
+            * if `None`
+
+          ContinueOnError (:class:`py:bool`):
+            Indicates to the server to continue sending responses
+            after errorresponse has been sent.
+            * if `True` the server may continue sending responses after
+              sending an error response.
+            * If `False` the server must not continue sending responses after
+              and error response is sent and must close the enumeration
+              sequence after sending any error response.
+            * if `None` this parameter is not sent to the WBEM server and the
+              WBEM server is to close the enumeration sequence upon error.
+
+          MaxObjectCount (term: `Uint32`)
+            Defines a value that sets the maximum number of instances the
+            WBEM server can return for this request. any Uint32 number including
+            zero is valid.
+            * If non-zero `Uint32` the WBEM server must return no more than the
+              number of instances defined by this value.
+            * If zero the WBEM server must return zero instances.  This may
+              be used by a client to restart the timeout.
+            * If `None`  a value of zero is sent to the WBEM server.
+
+          :Returns:
+
+            A tuple containing:
+            * A list of :class:`~pywbem.CIMInstance` objects
+              that are representations of the enumerated instances.
+            * instance of enumeration_context which contains
+              information about the state returned by the successfully
+              completed operation. This object may be queried with
+              the more() method to determine if this response
+              terminates the enumeration sequence.
+
+        :Exceptions:
+
+            See the list of exceptions described in `WBEMConnection`.
+        """
+
+        if namespace is None:
+            namespace = self.default_namespace
+
+        result = self._imethodcall(
+            'OpenEnumerateInstances',
+            namespace,
+            ClassName=CIMClassName(ClassName),
+            LocalOnly=LocalOnly,
+            DeepInheritance=DeepInheritance,
+            IncludeQualifiers=IncludeQualifiers,
+            IncludeClassOrigin=IncludeClassOrigin,
+            PropertyList=PropertyList,
+            FilterQueryLanguage=FilterQueryLanguage,
+            FilterQuery=FilterQuery,
+            ContinueOnError=ContinueOnError,
+            MaxObjectCount=MaxObjectCount,
+            pull_op=True,
```

Comment:
I don't like the pull_op arg and suggest we get rid of it. The method needs to deal with all kinds of return  values already.